### ISSUE TITLE
creation of variations is buggy when variations have special name

### ISSUE THREAD

**Issue by kinglouie** [jtl-software/connector-woocommerce3#7]
I have found a weird issue, when I create product variations in JTL by the attribute name "Name" the variations are not visible in the woocommerce frontend and backend, however in the B2B market plugin I can see that there are variations created for which I can enter special prices. 
When I delete the variations in JTL, rename the attribute to something else and recreate them, the sync works as expected.

Am on the latest connector version and Woocommerce 5.0

Screenshot for how to reproduce
![2021-03-04-083734_275x108_scrot](https://user-images.githubusercontent.com/8280409/109927900-24a31080-7cc5-11eb-929d-54c6dd316865.png)

**Comment by kinglouie** [comment]
I think I also found the root issue: It is because woocommerce does not allow to create a product taxonomy where the slug is "name". When I manually create a taxonomy i can sucessfully add a taxonomy called "Name" when I set the slug to something like "name1".
I think it would be nice if the connector plugin either reports an error if the taxonomy could not created or even better it would detect if the default sanitized slug is rejected and adds something like "-1" so it isn't rejected.
